usrsctplib/netinet/sctputil.c

```diff
@@ -4604,7 +4606,10 @@ sctp_abort_association(struct sctp_inpcb *inp, struct sctp_tcb *stcb,
 	                vrf_id, port);
 	if (stcb != NULL) {
 		/* We have a TCB to abort, send notification too */
-		sctp_abort_notification(stcb, 0, 0, NULL, SCTP_SO_NOT_LOCKED);
+		/* Read the cause code from the abort chunk */
+		cause = mtod(op_err, struct sctp_gen_error_cause *);
```

Comment:
This results in dereferencing a `NULL` pointer, if `op_err` is `NULL`. So I suggest to use something like
```
if (op_err != NULL) {
    cause = mtod(op_err, struct sctp_gen_error_cause *);
    cause_code = ntohs(cause->code);
} else {
    cause_code = 0;
}